rust/operator/src/config.rs

```diff
@@ -0,0 +1,212 @@
+use stackable_druid_crd::DruidRole;
+use std::collections::BTreeMap;
+
+pub fn get_jvm_config(role: &DruidRole) -> String {
+    let common_props = "
+    -server
+    -Duser.timezone=UTC
+    -Dfile.encoding=UTF-8
+    -Djava.io.tmpdir=/stackable/tmp/druid
```

Comment:
That folder doesn't exist and it causes trouble. Remove this option completely or use the system /tmp:

```suggestion
    -Djava.io.tmpdir=/tmp
```